components/layouts/dashboard/dashboardDrawer/dashboardDrawerItem/DashboardDrawerItem.mocks.ts

```diff
@@ -0,0 +1,21 @@
+// import { IDashboardDrawerItem } from './DashboardDrawerItem';
+// import AccessTimeIcon from '@mui/icons-material/AccessTime';
+// import AssignmentIcon from '@mui/icons-material/Assignment';
+// import {createElement} from "react";
+
+export {};
+
+//
+// const base: IDashboardDrawerItem = {
+//     text: "Drawer Item",
+//     icon: createElement(AccessTimeIcon),
+//     drawerOpen: true,
+//     href: "",
+//     parentText: "Parent Text",
+//     parentIcon: createElement(AssignmentIcon),
+// };
```

Comment:
There is prob still a way to do this so it can still pass lint and work with storybooks you might have to edit DashboardDrawerItem.stories.tsx